docs/core/testing/unit-testing-mstest-sdk.md

```diff
@@ -0,0 +1,103 @@
+---
+title: MSTest SDK
+author: MarcoRossignoli
+description: Overview of MSTest.Sdk.
+ms.author: mrossignoli
+ms.date: 02/13/2024
+---
+
+# MSTest SDK
+
+## Overview
+
+`MSTest.Sdk` is an [MSBuild project SDK](/visualstudio/msbuild/how-to-use-project-sdk) for building MSTest apps.  
+It's possible to build an MSTest app without this SDK, however, the MSTest SDK is:
+
+* Clear and coincise letting you to focus on your tests without the noise of the test infrastructure.
+* Simple to configure.
+* The recommended target for most users.
+
+How to use the `MSTest.Sdk` in a project:
+
+  ```xml
+  <!-- 
+       You can set a different sdk version, 3.3.0 is an example
+       You can set the SDK version at solution level using the global.json
+       Follow this guide:
+       https://learn.microsoft.com/visualstudio/msbuild/how-to-use-project-sdk?#how-project-sdks-are-resolved
+  -->
+  <Project Sdk="MSTest.Sdk/3.3.0">
+    <PropertyGroup> 
+        <TargetFramework>net8.0</TargetFramework>
+    </PropertyGroup>
+    <!-- references to the code to test -->      
+  </Project>
+  ```
+
+When you `build` the project all the needed components will be restored and installed using the standard NuGet workflow set by your project. Usually the components are downloaded from the official NuGet feed unless set differently.
```

Comment:
I would drop the last part personally but let's gather some other opinions

```suggestion
When you `build` the project all the needed components will be restored and installed using the standard NuGet workflow set by your project.
```

Reply:
I think it's fine it's not super useful info.